src/containers/Tabs/Account.tsx

```diff
@@ -42,19 +44,22 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
 
     const { drizzle } = drizzleReactHooks.useDrizzle();
 
-    const { addAccount, removeAccount } = drizzle.contracts.AccountRules.methods;
+    const { addAccount, removeAccount } = drizzle.contracts.AccountRules.methods as { 
+        addAccount: (value: string) => TransactionObject<any>, 
```

Comment:
rather than parameterising these over any either parameterise them over never to reflect we're not using the value. From whats been seen logging the paramterised value it doesn't line up to any known types in our ecosystem.